specs/tinted8/builder.md

```diff
@@ -0,0 +1,308 @@
+# Tinted8 Builder Guidelines
+
+**Version 0.1.0** The latest version of this spec can be obtained from
+[tinted-theming/specs/tinted8/builder](https://github.com/tinted-theming/home/blob/main/specs/tinted8/builder.md)
+
+## Introduction
+
+Builders convert a Tinted8 scheme into color data for templates to generate
+themes.
+
+## Inputs
+
+### Scheme Inputs
```

Comment:
On a side note, we may want to start defining the "scheme format" and the variables each builder exports differently.

I believe most of this has been copied from the base16 one.

Reply:
I forgot to add some of the inputs there (they were mentioned throughout the styling.md doc though) just pushed a commit. Base16 and 24 have 7 root scheme properties available, while there are now root 13 properties available here. What do you mean by defining the scheme format and and the vars they export differently?